app/src/main/java/org/sopt/dosopttemplate/LoginActivity.kt

```diff
@@ -0,0 +1,124 @@
+package org.sopt.dosopttemplate
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.inputmethod.InputMethodManager
+import androidx.activity.result.ActivityResult
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.dosopttemplate.Model.User
+import org.sopt.dosopttemplate.databinding.ActivityLoginBinding
+import org.sopt.dosopttemplate.util.getParcelable
+import org.sopt.dosopttemplate.utilprivate.makeToast
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    private lateinit var resultLauncher: ActivityResultLauncher<Intent>
+    private lateinit var user: User
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        getIntentInfo()
+
+        checkLoginAvailable()
+
+        signUpBtn()
+
+        hideKeyboard()
+    }
+
+    private fun getIntentInfo() {
+        resultLauncher =
+            registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+                if (result.resultCode == Activity.RESULT_OK) {
+                    setIDPW(result)
+                }
+            }
+    }
+
+    private fun setIDPW(result: ActivityResult) {
+        user = result.data?.getParcelable("USER", User::class.java) ?: return
+
+        binding.etvLoginId.setText(user?.id)
+        binding.etvLoginPw.setText(user?.pw)
```

Comment:
텍스트 set 작업을 코틀린의  프로퍼티 접근 구문(property access syntax)을 사용하여 
```suggestion
        binding.etvLoginId.text = user?.id
        binding.etvLoginPw.text = user?.pw
```
이렇게 바꿀 수 있습니다. 좀 더 코틀린스럽게, 간결하고 직관적이고 가독성 좋게 바꾸는 방법이지요, 아마 안스에서 위와 같이 바꾸라고 제안을 하고 있을 겁니다 :)

Reply:
EditText인줄 몰랐네요 코드리뷰 취소!